src/layer/heatmap/HeatMapColoredTile.js

```diff
@@ -33,13 +33,16 @@ define([
      *  color strings.
      *  @param options.extendedWidth {Number} Optional. Minimal width that needs to be retrieved for colorization.
      *  @param options.extendedHeight {Number} Optional. Minimal height that needs to be retrieved for colorization.
+     *  @param options.opacity {Number} Optional. The opacity applied to the calculated colors. The opacity is between 0 and 1. If this value isn't
```

Comment:
Please specify what 0 and 1 are in terms of transparency. In addition, the default is not longer the one mentioned.